create-links-in-hooks.sh

```diff
@@ -0,0 +1,6 @@
+mklink ".git/hooks/pre-commit" "%~dp0pre-commit"
```

Comment:
Да. Со всем согласен.
На самом деле немного поспешил.
Отлаживал под Win GIT Bash
При переходе на Ubuntu Локально поправил,
но на Github не поменял

2016-04-21 17:53 GMT+03:00 Shenja Sosna <notifications@github.com>:

> In create-links-in-hooks.sh
> <https://github.com/xDrivenDevelopment/precommit1c/pull/80#discussion_r60594627>
> :
>
> > @@ -0,0 +1,6 @@
> > +mklink ".git/hooks/pre-commit" "%~dp0pre-commit"
>
> mklink тоже,
> Получение %~dp0 - текущего пути в bash отличается.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/xDrivenDevelopment/precommit1c/pull/80/files/aa1b1fb99f54486ae85a4b2c1366a4ea543d9a56#r60594627>
>